### ISSUE TITLE
Provide a better explanation to each PNV category on requirements and outcomes

### ISSUE THREAD

**Issue by mikeburg** [burningmantech/ranger-clubhouse-web#215]
The Clubhouse needs to do a better job on explaining, possibly on the home page, what is required, what the expectations, and what the outcomes will be for each category of PNV. 

For Auditors, the Clubhouse should make it very clear passing training is not a path to becoming a BRC Ranger in the current year. Note, VCs are particularly interested in this since every year a sizable portions of Auditors email the team asking about Alpha shifts thinking they were on a path to become a BRC Ranger.

For Prospectives, the Clubhouse should guide the PNV thru the various various steps required to becoming an Alpha. An explanation on the home page, and various blockers on the Scheduling page are not the best solutions. Perhaps a wizard style interface to shepherd the PNV thru the steps  could be beneficial. 

For Alphas,  stating a mentor shift is still required and ideally the Clubhouse could nag the Alpha to sign up if no Alpha shift was found in their schedule.

This should be an in-depth conversation between the Tech Team, VCs, and Mentors concerning this issue.

**Comment by rangerthreepio** [comment]
Note that Auditors are not PNVs. It might be more accurate to say Non-Ranger categories, because Non-Rangers, Auditors, Prospectives, and Alphas are all Non-Rangers.

It might be worth reviewing the use of "(NR)" in this process.

**Comment by wsanchez** [comment]
Related: #172

**Comment by rangerthreepio** [comment]
Copy this. We can throw some new VCs at this because YF and I tried to do
this in 2019, and Phil and I tried to do this in 2018 and 2017, and people
just do NOT read the clubhouse messages when they log in, or really, ever.
They generally assume they are Alphas all season, and when they find out
they aren't they point at the Clubhouse as if it led them astray. So yea,
new VCs can formulate a plan for this, we have Tech-minded, UX minded
people on the team this year.
